spec/index.adoc

```diff
@@ -1680,6 +1681,8 @@ The returned response (and the response provided to the `transform` function, if
 The `transform` function may, for example, transform the body in any way, add or remove headers, modify headers, etc.
 When the transform function was invoked due to a canister HTTP request, the caller's identity is the principal of the management canister.
 
+The size of the response (from the remote server if no `transform` function is specified or after `transform`) in Candid encoding (including `status`, `headers`, and `body`) must not exceed 2045952 (`2MiB - 50KiB`).
```

Comment:
I don't know what we usually do. I find it hard to tell.

Reply:
@Dfinity-Bjoern , can you please elaborate what are your thoughts ?

Reply:
I don't think we need this paragraph. 
 Does candid do any compression ? How much would the size of the http response would change after the candid encoding ?

Reply:
I would formulate this in the spirit that the transform should not produce data bigger than our consensus blocks otherwise the operation would fail without going much into candid or strict limits.